test/e2e-go/features/stateproofs/stateproofs_test.go

```diff
@@ -0,0 +1,710 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
```

Comment:
We now have two files
`stateproofs_test.go` - current file
`stateproof_test.go` - e2e tests for stateproof keys and keyreg transcation.
I think we can find a better file name for `stateproof_test.go`

Reply:
There's also a unit test file called `stateproof_test.go`